Streetcode/Streetcode.XIntegrationTest/ControllerTests/Media/AudioControllerTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.DTO.Media.Images;
+using Streetcode.XIntegrationTest.ControllerTests.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Json;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Streetcode.XIntegrationTest.ControllerTests.Media
+{
+    public class AudioControllerTests:BaseControllerTests, IClassFixture<CustomWebApplicationFactory<Program>>
+    {
+        string secondPartUrl = "/api/Audio";
+        public AudioControllerTests(CustomWebApplicationFactory<Program> factory) : base(factory)
+        {
+
+        }
+
```

Comment:
Just for future tests, before we run tests please make sure all needed data inserted to database